docs/source/algorithms/SpecularReflectionPositionCorrect-v2.rst

```diff
@@ -27,64 +28,85 @@ Usage
 
 .. testcode:: SpecularReflectionPositionCorrectPointDetector
 
+   print 'point-detector'
    polref = Load(Filename=r'POLREF00004699.raw', PeriodList=1)
    polref = polref[0]
 
    instr = polref.getInstrument()
-   print instr.getComponentByName('point-detector').getPos()
+   print 'Original position: ' + str(instr.getComponentByName('point-detector').getPos())
 
-   polref = SpecularReflectionPositionCorrect(polref, TwoTheta = 2*0.49, DetectorComponentName='point-detector')
-   instr = polref.getInstrument()
-   print instr.getComponentByName('point-detector').getPos()
+   polref_vert = SpecularReflectionPositionCorrect(polref, TwoTheta = 2*0.49, DetectorComponentName='point-detector', CorrectionType='VerticalShift')
+   instr = polref_vert.getInstrument()
+   print 'Vertical shift:    ' + str(instr.getComponentByName('point-detector').getPos())
+
+   polref_rot = SpecularReflectionPositionCorrect(polref, TwoTheta = 2*0.49, DetectorComponentName='point-detector', CorrectionType='RotateAroundSample')
+   instr = polref_rot.getInstrument()
+   print 'Rotated:           ' + str(instr.getComponentByName('point-detector').getPos())
    
 Output:
 
 .. testoutput:: SpecularReflectionPositionCorrectPointDetector 
- 
-   [25.6,0,0.0444961]
-   [25.6,0,0.0444753]
+
+   point-detector
+   Original position: [25.6,0,0.0444961]
+   Vertical shift:    [25.6,0,0.0444753]
+   Rotated:           [25.6,0,0.0444753]
```

Comment:
Does this mean that in this case and for this angle, there's no difference between moving detectors vertically and rotating them?